tasks/install-docker-RHEL.yml

```
@@ -1,6 +1,6 @@
 ---
 - name: Ensure old versions of Docker are not installed.
-  package:
+  yum:

```

```suggestion
  dnf:
```
Логичнее перейти на dnf, т.к. yum [поддерживает только Python 2](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#synopsis).